src/styles/_catalog.scss

```diff
@@ -19,4 +20,40 @@
   &__card:not(:nth-child(4n)) {
     margin-right: 48px;
   }
+
+  @media (max-width: 488px) {
+    & {
+      display: grid;
+      grid-template-columns: repeat(1, 200px);
+      gap: 48px;
+      justify-content: center;
+    }
+  }
+
+  @media (min-width: 488px) {
+    & {
+      display: grid;
+      grid-template-columns: repeat(2, 200px);
+      gap: 48px;
+      justify-content: center;
+    }
+  }
+
+  @media (min-width: 768px) {
+    & {
+      display: grid;
+      grid-template-columns: repeat(3, 200px);
+      gap: 48px;
+      justify-content: center;
+    }
+  }
+
+  @media (min-width: 1024px) {
+    & {
+      display: grid;
+      grid-template-columns: repeat(4, 200px);
+      gap: 48px;
+      justify-content: center;
+    }
```

Comment:
These parameters 
display: grid;
gap: 48px;
justify-content: center; 

are the same and you already have them in .catalog, and it is not necessary to repeat them in  @media.

so in @media you can just change grid-template-columns: